File: src/main/java/um/haberes/report/model/haberes/core/AntiguedadDto.java
package um.haberes.report.model.haberes.core;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class AntiguedadDto {

    private Long antiguedadId;

    private Long legajoId;

    private int anho = 0;

    private int mes = 0;

    private int mesesDocentes = 0;

    private int mesesAdministrativos = 0;

    private PersonaDto persona;

}

File: src/main/java/um/haberes/report/kotlin/dto/haberes/core/AntiguedadDto.kt
package um.haberes.report.kotlin.dto.haberes.core

data class AntiguedadDto(

    var antiguedadId: Long? = null,
    var legajoId: Long? = null,
    var anho: Int = 0,
    var mes: Int = 0,
    var mesesDocentes: Int = 0,
    var mesesAdministrativos: Int = 0,
    var persona: PersonaDto? = null

)
